bors.toml

```diff
@@ -0,0 +1,23 @@
+# List of commit statuses that must pass on the PR commit when it is r+-ed.
+# (listed in same order as in the repo's branch protection settings)
+pr_status = [
+ "python-tests",
+ "tools-build",
+ "check-labels",
+ "static-tests",
+ "check-commits (commit-msg)",
+ "check-commits (pr-check)",
+ "Murdock",
+ ]
+
+# List of commit statuses that must pass on the merge commit before it is
+# pushed to master.
+status = [ "Murdock" ]
+
+# Number of project members who must approve the PR (using GitHub Reviews)
+# before it is pushed to master.
+# This is maybe not necessary with the check-labels action.
```

Comment:
Yes it is. When there is no review, but no special label set, the check-labels check will succeed :-).

Reply:
(also it might be good to have it as a fallback in general)

Reply:
my thoughts exactly. I'll update the comment.